package.json

```diff
@@ -120,6 +121,7 @@
     "ufo": "0.7.11",
     "unfetch": "4.2.0",
     "url-parse": "1.5.10",
+    "vue-chart-3": "3.1.8",
```

Comment:
note: `vue-chart-3` appears to be in maintenance mode[^1]. I think that we should reconsider adding it as a dependency to Dashboard:

> I'm joining [vue-chartjs](https://github.com/apertureless/vue-chartjs) as a core maintainer so this package will be kept just for those who already use it. I'm encouraging new users to use vue-chartjs instead

[^1]: https://github.com/victorgarciaesgi/vue-chart-3?tab=readme-ov-file#im-joining-vue-chartjs-as-a-core-maintainer-so-this-package-will-be-kept-just-for-those-who-already-use-it-im-encouraging-new-users-to-use-vue-chartjs-instead

Reply:
`vue-chart-3` is a temporary solution, we need to use a canvas + chart.js directly or enhance our Doughnut chart to support multiple colors: `shell/components/graph/Circle.vue` in the final version.